workshops/3D Heatmap/README.md

```diff
@@ -0,0 +1,202 @@
+
+You need to set up a repl account. you can run this program on your Python idle too. It requires some basic knowledge of programming. I have designed this workshop sweet and simple for you. It will just take 15-20 minutes to complete the workshop. 
+
+## Setting up
+
+- Repl is an online and instant development environment to learn, build, collaborate, and host your project. Show you don’t have to waste time while setting up a development environment. Creating an account on repl is super easy.
+
+[Create a repl](https://repl.it/signup)
+
+
+![repl website](https://cloud-73h0sldam.vercel.app/0screenshot_2020-12-25_at_23.03.53.png)
+
+- create a repl by clicking **+ New repl** and choose programming language **PYTHON** and named it as you want and click **create repl**
+
+- Download the datasets from a simple click and Drag and drop them to the folder in which you are working on repl.
+[volcano.csv](https://cloud-8ycpvzexa.vercel.app/0volcano.csv)
+
+
+- Rename the file as volcano.csv and place it in the repl folder in which you are working.
+
+- Setup is completed ✅.
+
+**Advantage of working on repl**
+
+*when we program in repl it automatically imports the libraries and dependencies with the simple command. So, You just need a repl account or any python idle to get started.*
+
+![Library](https://cloud-1th3ydnib.vercel.app/1workshop_library.gif)
+
+## Pandas🐼
+Pandas is a popular Python-based toolkit. It presents a diverse range of utilities like converting an entire data table into a NumPy matrix array and much more. This makes pandas a trusted ally in data science and machine learning.
+
+## Numpy🔢
+NumPy is a library used for working with arrays. It is used to perform a large set of mathematical operations on an array. An array is a data structure consisting of a collection of elements, each identified by at least one array index or key.
+
+
+## Matplotlib📊
+A picture is worth a thousand words, and with Python’s matplotlib library, fortunately, takes far less than a thousand words of code to create a production-quality graphic.
+
+## Toolkits🧰
+Toolkits are collections of application-specific functions that extend Matplotlib.
+
+## Seaborn䷀
+Seaborn library is one of the rarest earth metals. If you are a science student you will get...😄. Seaborn, provides a variety of visualization patterns. It uses fewer syntax and has interesting default themes.
+Great Right!😲
+
+# Step 1 Making 3D Plane
+
+Plotting 3-D Axis
+Importing libraries
+```python
+from mpl_toolkits.mplot3d import Axes3D
+import matplotlib.pyplot as plt
+import pandas as pd
+import seaborn as sns
+```
+The first step is to create 3D Axes, and then plot any of the 3D graphs. We pass ```projection='3d'``` to plt.axes, This is the empty canvas that we'll be painting.
+
+```python
+fig = plt.figure()
+ax = plt.axes(projection='3d')
+plt.show()
+```
+![3D Plane](https://cloud-a08hob7s0.vercel.app/1workshop2.gif)
+
+# 2. Plotting Graph
+
+Plotting 3-D a Graph
+```python
```

Comment:
why are you importing again this may be kind of confusing

Reply:
👍🏻